credentialdigger/client.py

```diff
@@ -1037,22 +1251,24 @@ def update_similar_snippets(self,
         int
             The number of similar snippets found and updated
         """
-
-        discoveries = self.get_discoveries(repo_url, file_name)
-        model = build_embedding_model()
-        """ Compute target snippet embedding """
-        target_snippet_embedding = compute_snippet_embedding(target_snippet,
-                                                             model)
+        disc = self.get_discoveries(repo_url, file_name)
+        discoveries = disc[1] if len(disc) == 2 else disc
```

Comment:
please put a comment because this instruction is not clear

Reply:
why should this repo have only 2 discoveries?

Reply:
That's the part where I'm handling the different output types of get_discoveries in the UI vs regular clients. In the UI classes, the discoveries are given by get_discoveries()[1] (while the discoveries count is givent by get_discoveries()[0]), so checking for output length allows to retrieve the discoveries in case the client is a UI client. I don't think that's a proper way of doing it, but otherwise I need to copy paste the code in the UI classes, which you don't seem to agree with.

Reply:
Ah ok now I understand. So, if this code is specific for the UI and doesn't work with the normal Client, we have to move it to ClientUI. And here keep a method working with the calls of the Client (if really needed).